src/lager_elogfmt_formatter.erl

```diff
@@ -21,10 +21,16 @@ format(Msg, Config, _Color) ->
 format(Msg, Config) ->
     App = proplists:get_value(app, Config),
     Defaults = proplists:get_value(defaults, Config, []),
+    Meta0 = meta(Msg, App),
+    Meta =
+        case proplists:get_value(strip_pid, Config, true) of
+            true -> lists:keydelete("pid", 1, Meta0);
+            false -> Meta0
+        end,
     Props = filter_undefined([{"app", App},
```

Comment:
Thanks, that's going into the right direction. I'd propose to change `filter_undefined` into `filter` or something and strip the pid there.